PostHog/Replay/PostHogSessionReplayConfig.swift

```diff
@@ -18,6 +18,17 @@
         /// Default: true
         @objc public var maskAllImages: Bool = true
 
+        /// Enable masking of all sandboxed system views
+        /// These may include UIImagePickerController, PHPickerViewController and CNContactPickerViewController
+        /// Experimental support
+        /// Default: true
+        @objc public var maskSandboxedViews: Bool = true
```

Comment:
```suggestion
        @objc public var maskAllSandboxedViews: Bool = true
```

just so it keeps the pattern from others.
we may introduce more granular approaches in the future such as a list of types